nflow_logging.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"github.com/Sirupsen/logrus"
+	"github.com/sirupsen/logrus"
```

Comment:
This is important as the maintainer of logrus changed their handle a while back, which may cause nflow to fail to build if newer tools have the new name as their dependency.